package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bfx-api-client-generator",
+  "version": "0.0.1",
+  "main": "src/generator.js",
+  "author": "hems <hems.inlet@gmail.com>",
+  "contributors":[],
```

Comment:
Missing space before `[]` (`npm i` corrects it & it causes a new diff)

Reply:
+1